<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateRangeDialog</class>
 <widget class="QDialog" name="DateRangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>173</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>232</width>
    <height>173</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>232</width>
    <height>173</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>请选择日期</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icon.qrc">
    <normaloff>:/images/clipboard.png</normaloff>:/images/clipboard.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>起始日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>结束日期：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="endDateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate_PushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_PushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Icon.qrc"/>
 </resources>
 <connections/>
</ui>
